Documentation/contributing/testing/e2e.rst

```diff
@@ -656,6 +656,52 @@ periodically update images in a ``cron`` job::
 .. _test/packet/scripts/add_vagrant_box.sh:
    https://github.com/cilium/cilium/blob/master/test/packet/scripts/add_vagrant_box.sh
 
+Known Issues and Workarounds
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+VirtualBox hostonlyifs and DHCP related errors
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you see an error of this form:
+
+::
+
+    A host only network interface you're attempting to configure via DHCP
+    already has a conflicting host only adapter with DHCP enabled. The
+    DHCP on this adapter is incompatible with the DHCP settings. Two
+    host only network interfaces are not allowed to overlap, and each
+    host only network interface can have only one DHCP server. Please
+    reconfigure your host only network or remove the virtual machine
+    using the other host only network.
+
+you can workaround this by trying the following steps:
+
+1) List all ``hostonlyif``:
+
+::
+
+    $ vboxmanage list hostonlyifs
+
+2) If found, delete them like so:
+
+::
+
+    $ vboxmanage hostonlyif remove <name>
+
+3) Try running the test suite again. If that doesn't work, repeat these steps
+and either reboot or reload all VirtualBox related kernel modules.
+
+4) If the above doesn't work, try the following:
+
+::
+
+    $ vboxmanage dhcpserver remove --netname HostInterfaceNetworking-vboxnet0
+
+Note that you might want to delete all occurrences of
+``HostInterfaceNetworking-vboxnetX`` (note the "X").
+
+5) Lastly, ensure Vagrant and VirtualBox are up-to-date.
+
```

Comment:
Do we still need this section now that we have the `dhcp_server_matches_config?` in the Vagrantfile?

Reply:
I think it could still be useful as a knowledge base. I recognize that it doesn't come off as such. Maybe I could reword it.

Reply:
> I think it could still be useful as a knowledge base. I recognize that it doesn't come off as such. Maybe I could reword it.

Maybe linking to docs (and SO answers if necessary) would be enough.

> Also, do we need to add the same workaround to the top-level Vagrantfile as well? I can't recall if I've run into this only in the test/ Vagrantfile or the top-level. FWIW, I only spin up the test/ Vagrantfile, but unsure of others' workflows.

I don't remember with which Vagrantfile I had this issue, but as far as I understand, it should affect both. I use both Vagrantfiles a lot.